.circleci/config.yml

```diff
@@ -57,7 +57,7 @@ references:
     NUM_NODES: 5
     HELM_REPOSITORY: codacy-unstable
     CODACY_URL: http://dev.k8s.dev.codacy.org
-    HELM_INSTALL_EXTRA_FLAGS: --set codacy-api.config.github.clientName=codacy-k8s-selfhosted-dev
+    HELM_INSTALL_EXTRA_FLAGS: --set codacy-api.config.github.clientName=codacy-k8s-selfhosted-dev --set global.github.app.appId=57855 --set global.github.app.privateKey=${GITHUB_PRIVATE_KEY_DEV} --set global.githubEnterprise.app.appId=2 --set global.githubEnterprise.app.privateKey=${GITHUB_ENTERPRISE_PRIVATE_KEY_DEV}
```

Comment:
--set global.github.app.appId=57855
--set global.githubEnterprise.app.appId=2

do these numbers have to be hardcoded here? can't we generate some random numbers in-place?

Reply:
We ca't generate the values, this are real app ids, what i can do is adding them as env vars if you want

Reply:
Got it. Yes I think it would be best to have those as env vars as well